.github/workflows/prepare_release.yml

```diff
@@ -0,0 +1,64 @@
+---
+
+name: Prepare Release
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        description: 'Module version to be released.'
+        required: true
+        type: string
+      allowed_owner:
+        description: 'Only allow this owner'
+        required: true
+        type: string
```

Comment:
Since it's triggered explicitly, is this really needed? The reason we have it in the release is that you can push tags to your own fork and you don't want it to start releasing from there.

Reply:
dropped it